client/my-sites/checkout/cart/secondary-cart.jsx

```diff
@@ -90,9 +94,18 @@ class SecondaryCart extends Component {
 					showCoupon={ true }
 				/>
 				<CartPlanDiscountAd cart={ cart } selectedSite={ selectedSite } />
+
+				{ isJetpackNotAtomic && <JetpackHeader /> }
```

Comment:
We probably just need the lighter and simpler [`JetpackLogo`](https://wpcalypso.wordpress.com/devdocs/design/jetpack-logo) here:

```suggestion
				{ isJetpackNotAtomic && <JetpackLogo full /> }
```

The header wraps up a bunch of conditional partner branding.

Reply:
@sirreal I tried using that one, but didn't see the `full` param so went for this one. Will fix soon!